04 Strategy Library/88 52-Weeks High Effect/02 Method.html

```diff
@@ -0,0 +1,99 @@
+<p>
+  <h4>Initialization of universe </h4>
+  We use coarse and fundamental selection as our universe selection. 
+  Here, <code>x.AssetClassification.MorningstarSectorCode</code> is used to assign each stock into different sectors. 
+  More available objects for use in QuantConnect Fine Universe filter can be found <a href="https://www.quantconnect.com/docs/data-library/fundamentals">here</a>.
```

Comment:
More _methods and tools that can be used_ in QuantConnect Fine Universe _filtering_ can be found